packages/error-parser/src/parsers/wrapCustomError.ts

```diff
@@ -0,0 +1,8 @@
+import { InternalError } from '../LiqualityErrors';
+
+export function wrapCustomError(customError: any) {
```

Comment:
this makes more sense as `createCustomError` 
when you read it in usage
`throw wrapCustomError(..)`
vs
`throw createCustomError(...)`

Reply:
It would be even better if this function didn't exist and instead `InternalError` took an optional `rawError` in constructor. 

Then usage is even more simple:
`throw new InternalError({ raw: CUSTOM_ERRORS.NotFound.LedgerTransportCreator })`

Reply:
by proxy i mean js proxy: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy

Reply:
Done